macvtap.go

```diff
@@ -73,6 +77,18 @@ func loadConf(bytes []byte) (*NetConf, string, error) {
 	return n, n.CNIVersion, nil
 }
 
+func getEnvArgs(envArgsString string) (*EnvArgs, error) {
```

Comment:
You can return `EnvArgs` directly. In case there is no envArgsString, just return EnvArgs{}. Then you don't need to compare envArgs with nil on line 227. Plus pointers do stuff on heap and I heard that's baaaad.

Reply:
Done.